### DISCUSSION TITLE
Airbnb Typescript support

### DISCUSSION THREAD

**Discussion by meunierfrederic** [super-linter/super-linter#1722] [Ideas]
Super-linter seems currently not supporting airbnb-typescript code style.
[npm page](https://github.com/iamturns/eslint-config-airbnb-typescript)
[GH repo](https://github.com/iamturns/eslint-config-airbnb-typescript)

Is it possible (and sufficient?) to append it in the `dependencies/package.json`?

> `"eslint-config-airbnb-typescript": "^12.3.1",`

Thanks

**Comment by meunierfrederic** [comment]
same question for standard-with-typescript code style: 

> `"eslint-config-standard-with-typescript": "^20.0.0",`

**Comment by ferrarimarco** [comment]
We added `eslint-config-airbnb-typescript` a while back, and [it looks like it's not being maintained anymore](https://github.com/iamturns/eslint-config-airbnb-typescript/issues/314) :(

We might have to remove it because it's holding back other dependency updates.

About `eslint-config-standard-with-typescript`: can you open a new feature request?
